corehq/apps/userreports/reports/view.py

```diff
@@ -104,7 +104,7 @@ def query_dict_to_dict(query_dict, domain):
     :param domain:
     :return: a dict
     """
-    request_dict = json_request(query_dict)
+    request_dict = json_request(query_dict, booleans_as_strings=True)
```

Comment:
Does this work when filtering both string and boolean typed columns? Is it worth writing some tests?